app/assets/javascripts/organizations.js

```diff
@@ -1,6 +1,13 @@
-$(document).pjax('a.organization-link', '#organization-content', {push: false} );
-$(document).pjax('a.edit-link-pjax', '#organization-content', {push: false} );
+$(document).pjax('a.organization-link', '#organization-content', {push: false );
+$(document).pjax('a.edit-link-pjax', '#organization-content', {push: false );
```

Comment:
Expected '}' to match '{' from line 2 and instead saw ')'.<br>Expected ')' and instead saw ';'.<br>Missing semicolon.